src/ServiceControl/Bootstrapper.cs

```diff
@@ -56,7 +50,7 @@ public Bootstrapper(Settings settings, BusConfiguration configuration)
         private void Initialize()
         {
             var loggingSettings = new LoggingSettings(settings.ServiceName);
-            ConfigureLogging(loggingSettings);
+            RecordStartup(loggingSettings);
```

Comment:
The settings class at this stage could already written entries to a log file, specifically when loading the [audit and error log queues](https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/Infrastructure/Settings/Settings.cs#L35). 

The only way to prevent that is to actually instantiate the `settings` variable after this initialize method has been called.

What about:


```CSharp
var loggingSettings = new LoggingSettings(host.ServiceName);
settings = settings ?? new Settings(host.ServiceName);
RecordStartup(loggingSettings);
```

Which also means the settings instantiation in the ctor then has to be removed.